README.md

```diff
@@ -2,9 +2,19 @@
 
 This project is part of the waarnemingen cluster.
 
-# VLOG - VRI Log - Verkeersregelinstallatie Log
-An app to store the VRI logs.
+## VLOG - VRI Log - Verkeersregelinstallatie Log
+An app to store the VRI logs. The ESB sents files to the POST endpoint provided by this app. This app loops over the 
+file and stores every line in a separate database record. 
+
+## Reistijden
+An app to store the reistijden of cars driving through Amsterdam. It's an endpoint to which XML files are posted.
```

Comment:
De app heet 'Reistijden v1', wat het bestaan van meerdere (eventueel toekomstige?) versies van de reistijden app impliceert. Misschien goed om daar iets over te zeggen in deze doc.

Reply:
Het is meer omdat ik in cmsa van alles versies krijg. Je kan die in dezelfde app zetten, maar dan kan je modelveranderingen niet doorvoeren en beide endpoints up houden. Daar gebruik ik dus een app per versie. Omdat een app later hernoemen wat geklooi met zich meebrengt heb ik deze eerste maar meteen v1 genoemd.